test/integration/document/Thumbnails.e2e.test.js

```diff
@@ -1,16 +1,27 @@
 // <reference types="Cypress" />
 describe('Preview Document Thumbnails', () => {
     const token = Cypress.env('ACCESS_TOKEN');
-    const fileId = Cypress.env('FILE_ID_DOC');
+    const fileId = Cypress.env('FILE_ID_DOC_LARGE');
     const THUMBNAIL_SELECTED_CLASS = 'bp-thumbnail-is-selected';
 
     /* eslint-disable */
```

Comment:
Do you recall why this is necessary? It's wrapping more functions now.

Reply:
Mainly to avoid the eslint errors around not having jsdoc for these helper functions. but I'm thinking it would be helpful to treat the test code as important as production code, so I'll remove these and add some jsdoc